### ISSUE TITLE
sbt-scalafmt 0.2.11 crashes with sbt-0.13.12 on scala 2.10.5 project

### ISSUE THREAD

**Issue by apivovarov** [olafurpg/scalafmt#373]
build.sbt has
```
lazy val scalaVersion = "2.10.5"
scalafmtConfig in ThisBuild := Some(file(".scalafmt"))
reformatOnCompileSettings
```
if I run sbt package it crashes
```
[info] Formatting 1 Scala source {file:/Users/.../}foobar ...
java.lang.NoSuchMethodError: scala.Option.contains(Ljava/lang/Object;)Z
	at org.scalafmt.cli.StyleCache$.getConfigForFile(StyleCache.scala:18)
	at org.scalafmt.cli.StyleCache$.getStyleForFile(StyleCache.scala:35)
	at org.scalafmt.cli.Scalafmt210.format(Scalafmt210.scala:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.scalafmt.sbt.HasScalaFmt.org$scalafmt$sbt$HasScalaFmt$$handleFile(HasScalaFmt.scala:85)
	at org.scalafmt.sbt.HasScalaFmt$$anonfun$writeFormattedContentsToFiles$1$$anonfun$apply$3.apply(HasScalaFmt.scala:54)
	at org.scalafmt.sbt.HasScalaFmt$$anonfun$writeFormattedContentsToFiles$1$$anonfun$apply$3.apply(HasScalaFmt.scala:54)
	at scala.collection.Iterator$class.foreach(Iterator.scala:727)
	at scala.collection.parallel.immutable.ParHashSet$ParHashSetIterator.foreach(ParHashSet.scala:76)
	at scala.collection.parallel.ParIterableLike$Foreach.leaf(ParIterableLike.scala:975)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply$mcV$sp(Tasks.scala:54)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply(Tasks.scala:53)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply(Tasks.scala:53)
	at scala.collection.parallel.Task$class.tryLeaf(Tasks.scala:56)
	at scala.collection.parallel.ParIterableLike$Foreach.tryLeaf(ParIterableLike.scala:972)
	at scala.collection.parallel.AdaptiveWorkStealingTasks$WrappedTask$class.compute(Tasks.scala:165)
	at scala.collection.parallel.AdaptiveWorkStealingForkJoinTasks$WrappedTask.compute(Tasks.scala:514)
	at scala.concurrent.forkjoin.RecursiveAction.exec(RecursiveAction.java:160)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

workaround is to use sbt-0.13.9

**Comment by olafurpg** [comment]
Duplicate of #190, which has been open since April. However, it seems like I just figured out a fix for it. Let me confirm.

**Comment by olafurpg** [comment]
Closing this as a duplicate. Pending fix in #374
